semantic_conventions/metrics/process-runtime-jvm-metrics.yaml

```
@@ -127,8 +127,10 @@ groups:
     metric_name: process.runtime.jvm.cpu.utilization
     brief: "Recent CPU utilization for the process."
     note: >
+      The value range is [0,1].
```

```suggestion
      The value range is [0.0,1.0].
```
I quite like how it is explained in the link you provided, specially that it's a double which is not obvious from the example here. Do you think it's obvious enough that people won't benefit from more details?